File: js-pr/56/javascript/code-quality-error-handling/subprocess_runner_38.js
// {fact rule=code-quality-error-handling@v1.0 defects=0}
function runCommand(command) {
    const { execSync } = require('child_process');
    try {
        return execSync(command, { encoding: 'utf8' });
    } catch (error) {
        console.error('Command execution error:', error.message);
        return null;
    }
}
// {/fact}

File: python-pr/52/python/code-quality-error-handling/subprocess_runner_38.py
# {fact rule=code-quality-error-handling@v1.0 defects=0}
def run_command(command):
    import subprocess
    try:
        return subprocess.run(command, shell=True, check=True)
    except subprocess.CalledProcessError as e:
        print(f"Command error: {e}")
        return None
# {/fact}